pyproject.toml

```diff
@@ -9,3 +9,5 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 80
+
+DJANGO_SETTINGS_MODULE = "electionleaflets.settings.base"
```

Comment:
`DJANGO_SETTINGS_MODULE` under `[tool.black]` isn't doing anything. `tool.black.DJANGO_SETTINGS_MODULE` is not a setting of black. I realise this is a hangover from trying to get pytest to work, but we can delete it from here.

Reply:
This still isn't doing anything.

If (/when) we add a `[tool.pytest.ini_options]` section to `pyproject.toml` we would put this in `pyproject.toml` because `tool.pytest.ini_options.DJANGO_SETTINGS_MODULE` is a setting.

At the moment, we've got nothing that is looking at `pyproject.toml` for this setting